.github/workflows/image_test.yaml

```diff
@@ -0,0 +1,87 @@
+name: Images build series for Environment  
+
+on: 
+  push:
+    branches:
+      - 'dev_fission'
+  pull_request:
+    branches:
+      - 'dev_fission'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: docker login
+      env:
+        DOCKER_USER: "msat1234"   
+        DOCKER_PASSWORD: "Msat1234#" 
+        DOCKER_BUILDKIT: 1
+        BUILDX_PLATFORMS: linux/amd64,linux/arm64,linux/ppc64le,linux/s390x,linux/386,linux/arm/v7,linux/arm/v6 
```

Comment:
I don't think we support all arch here. Only `linux/amd64,linux/arm64` and I think `linux/arm/v7`